data/formats-data.js

```diff
@@ -3774,7 +3781,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	shinx: {
-		randomBattleMoves: ["wildcharge", "icefang", "firefang", "crunch"],
+		randomBattleMoves: ["wildcharge", "icefang", "firefang", "voltswitch"],
```

Comment:
Facade for Guts

Reply:
id rather intimidate pivot

Reply:
shinx hits 14 speed (normally) and 16 attack (normally with ev investment), and with stuff like mienfoo spamming high jump kick, even after intimidate, it's doing a lot.  The only mon that really uses intimidate is growlithe in LC, and maybe scraggy to help set up. LC generally prefers switching to a hard counter, due to damage rolls.

Reply:
The ability to hard counter things here will likely fall but I still see your point, it can come in